### ISSUE TITLE
Spellchecker not working on Windows in 5.4.0

### ISSUE THREAD

**Issue by rogergbh** [getferdi/ferdi#220]
After update spell check isn't working for any service. Toggle is on. 
Same after a restart and toggle on and off. 

I reverted back to 5.3.3 and spell check works as expected.

**Comment by vantezzen** [comment]
We have switched to a different spell checker in 5.4.0 so that will be the cause of this. What OS are you running?

**Comment by rogergbh** [comment]
oops, sorry!
Win 10 Pro build 19013

**Comment by rogergbh** [comment]
> We have switched to a different spell checker in 5.4.0 so that will be the cause of this. What OS are you running?

oops, sorry!
Win 10 Pro build 19013

**Comment by Gaboris** [comment]
I'm using a Hungarian Win7 64bit and the spellchecker is not running for me either. :(

**Comment by vantezzen** [comment]
@Gaboris Could you open the service developer console to see if there are any errors?

**Comment by Gaboris** [comment]
Oh, right... Well there are 9 errors, but most are about the login not working sooo let me restart...

Darn it. Nothing with a clean start.
You have any keywords I could filter for? I tried spell, but nothing came up in the full list.

There is something under Verbose for the React Devtool, but I'm not sure if that has anything to do with the spellchecker.

**Comment by ahobsonsayers** [comment]
Thought id drop in and mention, I'm also having this issue

**Comment by bddckr** [comment]
Same here. Windows 10, testing in Slack right now and getting this error on load of the service:
```
C:\Users\bddckr\AppData\Local\Programs\ferdi\resources\app.asar\webview\spellchecker.js:1 Error: document.body is null, if you're calling this in a preload script you need to wrap it in a setTimeout
    at SpellCheckHandler.attachToInput (C:\Users\bddckr\AppData\Local\Programs\ferdi\resources\app.asar\node_modules\electron-spellchecker\src\spell-check-handler.js:214)
    at initialize (C:\Users\bddckr\AppData\Local\Programs\ferdi\resources\app.asar\webview\spellchecker.js:1)
    at _temp.initialize (C:\Users\bddckr\AppData\Local\Programs\ferdi\resources\app.asar\webview\recipe.js:1)
    at new _temp (C:\Users\bddckr\AppData\Local\Programs\ferdi\resources\app.asar\webview\recipe.js:1)
    at Object.<anonymous> (C:\Users\bddckr\AppData\Local\Programs\ferdi\resources\app.asar\webview\recipe.js:1)
    at Object.<anonymous> (C:\Users\bddckr\AppData\Local\Programs\ferdi\resources\app.asar\webview\recipe.js:2)
    at Module._compile (internal/modules/cjs/loader.js:786)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:798)
    at Module.load (internal/modules/cjs/loader.js:645)
    at Function.Module._load (internal/modules/cjs/loader.js:560)
```

**Comment by kytwb** [comment]
Adding a setTimeout as they do in [electron-spellchecker exemple](https://github.com/electron-userland/electron-spellchecker/blob/7227d61415c47dfa247828da3b62bbb3330095aa/example/preload.js#L8).

**Comment by kytwb** [comment]
The potential fix previously pushed has been shipped with [v5.4.1-beta.1](https://github.com/getferdi/ferdi/releases/tag/v5.4.1-beta.1). Feedback welcome!

**Comment by bddckr** [comment]
I can confirm 5.4.1-beta.1 solves this issue in Slack services at least! No more errors in the dev console either. Thanks for the fix!
